cmd/influxd/config.go

```diff
@@ -251,6 +252,43 @@ func ParseConfig(s string) (*Config, error) {
 	return c, nil
 }
 
+type Graphite struct {
+	Addr          string `toml:"address"`
+	Database      string `toml:"database"`
+	Enabled       bool   `toml:"enabled"`
+	Port          uint16 `toml:"port"`
+	Protocol      string `toml:"protocol"`
+	NamePosition  string `toml:"name-position"`
+	NameSeparator string `toml:"name-separator"`
+}
+
+// ConnnectionString returns the connection string for this Graphite config in the form host:port.
+func (g *Graphite) ConnectionString(defaultBindAddr string) string {
+	var addr string
+	var port uint16
```

Comment:
nit: can you move these declarations down to the sections of code that use them?